build.gradle

```diff
@@ -30,5 +30,5 @@ task clean(type: Delete) {
 ext {
     supportLibraryVersion = '27.1.1'
     butterknifeVersion = '8.8.1'
-    daggerVersion = '2.16'
-}
\ No newline at end of file
+    daggerVersion = '2.19'
+}
```

Comment:
This change is not required.

Reply:
may i ask why?

Reply:
You just have to change the version number but you have changed the last line as well which I think is not required, it not major though it would be good if it gets fixed.

Reply:
oh yes... i get it